src/javascript-wrapper.js

```diff
@@ -64,26 +36,19 @@ var fRunUpdater = function(updater, updaterFormals, vni) {
      }).then(function(results) { 
          // console.log('updater returned results: ',results);
 
-         if (! _.isUndefined(results)) { 
+         if (! _.isUndefined(results)) {
              // Got some results back from updater
              // If the updater returned anything, set the output state with it
-             var newState = vni.outputState();
-             if (newState.lm === oldOutputStateLm || 
-                 _.isUndefined(oldOutputStateLm) && ! _.isUndefined(newState.lm)) { 
+             var newStateLm = vni.outputState().lm;
+             if (newStateLm === oldOutputStateLm ||
+                 _.isUndefined(oldOutputStateLm) && ! _.isUndefined(newStateLm)) {
```

Comment:
It does not on the first request.  I added this check precisely because it doesn't.